build.ps1

```diff
@@ -210,24 +220,33 @@ if (!$SkipBuild)
             Remove-Item -Force -Recurse "$repoPath\obj\"
         }
 
-        # NuGet restore disregards platform/configuration
         # dotnet restore defaults to Debug|Any CPU, which requires the /p:platform specification in order to replicate NuGet restore behavior.
-        $restoreArgsInit = "restore $repoPath\InstrumentationEngine.sln --configfile $repoPath\NuGet.config"
-        $restoreArgs = @(
-            "$restoreArgsInit /p:platform=`"x86`""
-            "$restoreArgsInit /p:platform=`"x64`""
-            "$restoreArgsInit /p:platform=`"Any CPU`""
-        )
-        Invoke-ExpressionHelper -Executable "dotnet" -Arguments $restoreArgs -Activity 'dotnet Restore Solutions'
+        $dotnetRestoreArgs = "restore `"$repoPath\InstrumentationEngine.sln`" --configfile `"$repoPath\NuGet.config`" /p:platform=`"Any CPU`""
```

Comment:
This is now different than what is at line ~270 (other than the solution name). Why is that?

Reply:
These changes to the build scripts are to mirror the pipeline build. It also looks like the package solution is only using nuget restore, not dotnet restore so I'll modify this accordingly